### ISSUE TITLE
Lint does not recognise one-element tuples

### ISSUE THREAD

**Issue by Michael-Klassen** [tonyhffong/Lint.jl#110]
```julia
julia> lintstr("""
       day = 1
       day, = (2,3)
       """)
1-element Array{Lint.LintMessage,1}:
 none:2 [               ] WARN   Previously used day has apparent type Int64, but now assigned Tuple{Int64,Int64}
```

**Comment by tonyhffong** [comment]
thanks for the report. Fixed.
